src/www/ui/api/Helper/DbHelper.php

```diff
@@ -310,7 +310,7 @@ public function getJobs($id = null, $limit = 0, $page = 1, $uploadId = null)
 
     $params = [];
     $statement = __METHOD__ . ".getJobs";
-    $countStatement = __METHOD__ . ".getJobCount";
+    $count  Statement = __METHOD__ . ".getJobCount";
```

Comment:
Uhho. Please fix the variable name.

Reply:
Thank you @GMishx , the naming is now fixed.